### PR TITLE
add new package source and update readme

### PR DESCRIPTION
**Pull request by aacebo**
Add github package source and instructions on how to install published package versiosn.

https://github.com/microsoft/teams-ai/issues/360

### CODE REVIEW

File: dotnet/samples/01.messaging.echoBot/Nuget.Config [hunk 1]
```diff
@@ -2,6 +2,14 @@
 <!-- Temporary local nuget source before public release -->
 <configuration>
 	<packageSources>
-		<add key="Local Package" value="./LocalPkg" />
+		<clear />
+        <add key="Local Package" value="./LocalPkg" />
+		<add key="github" value="https://nuget.pkg.github.com/microsoft/index.json" />
```
Comment (swatDong) [inline dotnet/samples/01.messaging.echoBot/Nuget.Config:7]: It seems the indent is a bit weird. Is that because mixed tab ("/t") and space (" ")?

**Comment by swatDong** [comment]
Update all *.csproj to reference the alpha version. E.g., *\<PackageReference Include="Microsoft.TeamsAI" Version="**1.0.0-\***" /\>* ?

File: dotnet/samples/01.messaging.echoBot/Nuget.Config (hunk 1 above)
Comment (aacebo) [reply dotnet/samples/01.messaging.echoBot/Nuget.Config:7]: good catch, ill fit that

Comment (aacebo) [reply dotnet/samples/01.messaging.echoBot/Nuget.Config]: I think I resolved it, I also added a editoconfig to each sample to try and make sure it doesn't happen again

**Comment by aacebo** [comment]
> Update all *.csproj to reference the alpha version. E.g., _<PackageReference Include="Microsoft.TeamsAI" Version="**1.0.0-***" />_ ?

updated to reference the latest deployed version in our github package source.

**Review by swatDong — APPROVED** [review]

**Pull request merged**